tools/python/checkupstreamversion.py

```diff
@@ -44,4 +44,5 @@
     print(
         f"Local version {local_version} does not match upstream version {upstream_version}"
     )
+    os.environ["LIBPHONENUMBER_UPSTREAM_VERSION"] = upstream_version
```

Comment:
I don't think this will work &ndash; IIRC environment variables are only visible to child processes, not parents.